src/file/file.grpc.ts

```diff
@@ -45,7 +45,8 @@ export class FileMethods {
   public static async GetFilesByFolder(call: any, callback: any): Promise<{ files: ResFile[] }> {
     const folderID: string = call.request.folderID;
     const ownerID: string = call.request.ownerID;
-    const files: IFile[] = await FileService.getFilesByFolder(folderID, ownerID);
+    const query: string = call.request.query || '{}';
```

Comment:
We don't pass a string "JSON Query", we just pass any subset of the File model, aka `Partial<IFile>`.

Reply:
```
const file: IFile | { [key: string]: any } = {};
	const newFile: Partial<IFile> = {};
	for (const prop in file) {
		if (file[prop]) {
			newFile[prop] = file[prop];
		}
	}
```